### PR TITLE
Re-introduce field formats on ECS fields.

### PR DESCRIPTION
**Pull request by webmat**
They were accidentally removed by #11150.

### CODE REVIEW

**Comment by webmat** [comment]
Since this reintroduces field formats exactly as they were before #11150, I don't think this requires a changelog.

**Comment by webmat** [comment]
[Error](https://beats-ci.elastic.co/job/elastic+beats+pull-request+multijob-linux/6025/beat=metricbeat,label=linux-immutable/console) seems unrelated, in Metricbeat test suite:

```
22:20:14 --- FAIL: TestData (2.15s)
22:20:14 	elasticsearch_integration_test.go:109: getting elasticsearch version Get http://elasticsearch:9200/: dial tcp: lookup elasticsearch on 127.0.0.11:53: no such host
```

**Comment by webmat** [comment]
jenkins, test this

File: libbeat/_meta/fields.ecs.yml [hunk 1]
```diff
@@ -1807,6 +1813,7 @@
     - name: bytes
       level: core
       type: long
+      format: bytes
       description: Bytes sent from the source to the destination.
       example: 184
     - name: domain
```
Comment (simitt) [inline libbeat/_meta/fields.ecs.yml]: Looks like you missed:
* [http.request.body.bytes](https://github.com/elastic/beats/blob/5a6433488c2e99abaebf7d4f81008375c32b7321/libbeat/_meta/fields.ecs.yml#L1079)
* [http.request.bytes](https://github.com/elastic/beats/blob/5a6433488c2e99abaebf7d4f81008375c32b7321/libbeat/_meta/fields.ecs.yml#L1090)
* [http.response.body.bytes](https://github.com/elastic/beats/blob/5a6433488c2e99abaebf7d4f81008375c32b7321/libbeat/_meta/fields.ecs.yml#L1110)
* [http.response.bytes](https://github.com/elastic/beats/blob/5a6433488c2e99abaebf7d4f81008375c32b7321/libbeat/_meta/fields.ecs.yml#L1121)

Comment (webmat) [reply libbeat/_meta/fields.ecs.yml]: Thanks, you're right. I'll go add them
🏃

**Comment by webmat** [comment]
Rebased on top of master. I think #11230 fixes the Metricbeat test flakiness

**Pull request merged**